src/ECModel.jl

```diff
@@ -673,7 +673,7 @@ function plot_sankey(ECModel::AbstractEC, sank_data::Dict)
         node_colors=sank_data["colors"],
         edge_color=:gradient,
         compact=true,
-        label_size=10,
+        label_size=label_size,

```

Comment:
Ok, but you need to add the label_size arbument at the function:
`plot_sankey(ECModel::AbstractEC, sank_data::Dict; label_size=10)`

This should be the last step maybe